bolero-engine/src/target_location.rs

```diff
@@ -28,20 +28,18 @@ pub struct TargetLocation {
 
 impl TargetLocation {
     pub fn should_run(&self) -> bool {
```

Comment:
Might be cleaner to duplicate this function instead since there is no common code between kani and not(kani)?

Reply:
sure